curriculum/superblock-structure/full-stack.json

```diff
@@ -547,16 +547,7 @@
           "dashedName": "security-and-privacy",
           "blocks": [
             { "dashedName": "review-security-and-privacy" },
```

Comment:
we decided yesterday that this will be removed too. 
This will be added in later once we figure out what we want to do with it.
Please remove the associated files for the security and privacy review

Reply:
this PR removes them https://github.com/freeCodeCamp/freeCodeCamp/pull/57051 @jdwilkin4 
Once this is merged I can fix the conflicts and then we should have only the correct blocks.

Reply:
Ok cool 👍